src/home/homework-luks.c

```diff
@@ -1901,27 +1903,31 @@ static int make_partition_table(
         if (r < 0)
                 return log_error_errno(r, "Failed to place partition at first free partition index: %m");
 
+        // Use same sector size as the fdisk context when converting to bytes
```

Comment:
coding style: in our tree we use `/* */` style coments for explanatory stuff, and `//` style comments for "fixme" style stuff. this here is explanatory, hence please change comment style.

(coding style is documented here: https://systemd.io/CODING_STYLE)